test/parallel/test-stream2-httpclient-response-end.js

```diff
@@ -11,7 +11,9 @@ const server = http.createServer(function(req, res) {
     let data = '';
     res.on('readable', common.mustCall(function() {
```

Comment:
the fact that it is currently passing leads me to believe there is a bug where the `null` readable event is not being fired